src/bms/command/minor_command.rs

```diff
@@ -143,7 +143,7 @@ pub struct StpEvent {
     pub duration: Duration,
 }
 
-/// MacBeat `#WAVCMD` event.
+/// `MacBeat` `#WAVCMD` event.
```

Comment:
MacBeat is a proper nourn, software name. It should not markup as an inline code.